### PR TITLE
rename package to io.tidb

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

File: flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public static DataType getFlinkType(org.tikv.common.types.DataType dataType) {
   /**
    * transform TiKV java object to Flink java object by given Flink Datatype
    *
-   * @param object TiKV java object
+   * @param object   TiKV java object
```
Comment (sunxiaoguang) [inline flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java:109]: There is an extra space here @marsishandsome

Comment (marsishandsome) [reply flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java:109]: after import `intellij-java-google-style.xml` to Intellij, `align parameter descriptions` is set to true.

image

**Comment by humengyu2012** [comment]
LGTM

File: flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java (hunk 1 above)
Comment (sunxiaoguang) [reply flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TypeUtils.java:109]: Hm, good to know it.

**Comment by sunxiaoguang** [comment]
LGTM

**Review by sunxiaoguang — APPROVED** [review]

**Pull request merged**